src/event_imp.rs

```diff
@@ -1044,6 +1044,216 @@ fn test_debug_ready() {
     assert_eq!("Writable", format!("{:?}", Ready::writable()));
 }
 
+
```

Comment:
Double empty line.

Reply:
I _think_ `rustfmt` will take care of this. I would recommend running a `rustfmt` on this branch, it should hopefully make the merge easier.